src/System.Runtime/src/ApiCompatBaseline.txt

```diff
@@ -0,0 +1,4 @@
+MembersMustExist : Member 'System.Type.IsSecurityCritical.get()' does not exist in the implementation but it does exist in the contract.
```

Comment:
Didn't coreclr update bring these implementations? I'm not sure if you have Stephen changes but basically he merged #11300 which should have all of your reflection changes.